jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Why do this here instead of `package.json`?

Reply:
I think this is just a matter of consistency - with all the other configs moving out of `package.json`, I think it would be unexpected for this config to continue living there.